airguardian-core/drone/src/main/kotlin/systems/ajax/motrechko/airguardian/drone/application/util/BatteryCalculator.kt

```diff
@@ -1,6 +1,6 @@
-package systems.ajax.motrechko.airguardian.utils
+package systems.ajax.motrechko.airguardian.drone.application.util
```

Comment:
Now this class looks more like a service. It's not call to action, just informing you that it's more correct making the class `@Service`

Reply:
Do I need to annotate `@Service` or rewrite this class and move it to the application layer?